index.html

```diff
@@ -1671,33 +1669,57 @@ <h3>Choosing a definition of 'string'</h3>
+ 
+    <div class="req" id="char-string-uint8">
+        <p class="advisement"> Specify {{Uint8Array}} when working with byte sequences, such as for data that does not contain text, or for byte sequences representing text for which processing is never required (such as when copying buffers).</p>
+    </div>
+    
+    <p>Some protocols or formats do not distinguish between bytes and strings. For example, HTTP [[RFC9112]] says:</p>
+    
+    <p class="localdef">A recipient MUST parse an HTTP message as a sequence of octets in an encoding that is a superset of US-ASCII.</p>
+    
+    <div class="req" id="char_string_byte">
+	<p class="advisement">Specify {{ByteString}} in the rare cases where the specification needs to work with strings which are encoded using bytes and for which the conversion to or from Unicode would be inappropriate.</p>
 	<details class="links"><summary>explanations &amp; examples</summary>
-	<p>See also <a href="#char_choosing"></a>.</p>
+	   <p><a href="https://www.w3.org/TR/design-principles/#idl-string-types">IDL String Types</a> in <cite>Web Platform Design Principles</cite> [[DESIGN-PRINCIPLES]]</p>
 	</details>
-</div>
-
-<p>The type {{ByteString}} defines strings as sequences of bytes (octets). Interpretation of byte strings thus requires the specification of a <a>character encoding form</a>. UTF-8 is the preferred encoding for wire and document formats [[ENCODING]], but there is generally no reason to specify strings in terms of the underlying byte values.</p>
+    </div>
 
-<aside class="note">
-	<p>Specifications for document formats or protocols often deal with the specific byte values used for various fields or values or with the <a>character encoding</a> used for serializing the data. It is therefore tempting to specify a text field ("string") as a {{ByteString}} which uses the <a>UTF-8</a> <a>character encoding form</a>.</p>
-	
-	<p>It is preferable, however, to specify these fields as a {{DOMString}} (or, rarely, a {{USVString}}), since the data encoded into these fields must be serialized from and deserialized into in-memory string representations, such as the [[DOM]] or JavaScript strings or your platform's native Unicode string type.</p>
-</aside>
+    <p>{{ByteString}} isn’t a general-purpose string type. Frequently processing of these will be done by performing an [=isomorphic decode=] of the {{ByteString}} into an [=isomorphic string=] or by performing an [=isomorphic encode=] of such a string back into bytes [[INFRA]]. (It is also possible that the specification with work with the bytes directly.)</p>
```

Comment:
Per Web IDL a ByteString always maps to an isomorphic string. I suppose it's true that unusual processing might take place, but in all cases I can recall we leave it at that. And unusual processing might take place for all string types, but we're not calling that out either so it seems weird to treat this type so circumspect.